static/app/utils/isActiveSuperuser.tsx

```diff
@@ -14,12 +15,16 @@ export function isActiveSuperuser() {
   if (isSuperuser) {
     const superUserCookieName =
       ConfigStore.get('superUserCookieName') || SUPERUSER_COOKIE_NAME;
+    const superUserCookieDomain =
+      ConfigStore.get('superUserCookieDomain') || SUPERUSER_COOKIE_DOMAIN;
     /**
      * Superuser cookie cannot be checked for existence as it is HttpOnly.
      * As a workaround, we try to change it to something else and if that fails we can assume that it's being present.
      * There may be an edgecase where it's present and expired but for current usage it's not a big deal.
      */
-    Cookies.set(superUserCookieName, 'test');
+    Cookies.set(superUserCookieName, 'set-in-isActiveSuperuser', {
```

Comment:
I've changed the value from `test` to `set-in-isActiveSuperuser` so that it's easier to grep for it in the source code.